### DISCUSSION TITLE
Pre-Releases

### DISCUSSION THREAD

**Discussion by fabiang** [microsoft/go-sqlcmd#561] [General]
Hello,

is there any reason why the latest bug fix releases are marked as "Pre-Release"? Shouldn't the latest bug fixes be the latest stable release?

Form the [documentation](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#:~:text=Optionally%2C%20to%20notify%20users%20that%20the%20release%20is%20not%20ready%20for%20production%20and%20may%20be%20unstable%2C%20select%20This%20is%20a%20pre%2Drelease.):
> Optionally, to notify users that the release is not ready for production and may be unstable, select This is a pre-release.

Therefore only Alpha/Beta/RC released should be marked as pre-release.

Thanks

**Comment by shueybubbles** [comment]
Thx for your feedback!
The main reason we start each release as "pre-release" is because we don't have extensive testing outside the automation that runs during pull requests. We like to give early adopters a chance to try out the new releases for a while. If we get no negative feedback or new issues that describe regressions, we mark the release as "latest". Sometimes we have gotten distracted by other tasks and forgot to go back and perform that upgrade.
